src/librustdoc/html/static/js/search.js

```diff
@@ -1926,6 +1944,21 @@ class DocSearch {
             query.error = err;
             return query;
         }
+
+        function handleCrateFilters(elem) {
+            if (elem.typeFilter === TY_CRATE) {
+                query.filterCrates = elem.name;
+                return false;
+            }
+            return true;
+
+        }
+        const nonCrateElems = query.elems.filter(handleCrateFilters);
+        if (nonCrateElems.length !== query.elems.length) {
+            query.elems = nonCrateElems;
+            query.userQuery = query.elems.join(", ");
```

Comment:
right, i think i added that before realizing the limitation of needing the comma.  it won't break everything, since the search algo also checks the `name` field of the elements but it definitely isn't working as intended, will remove.